spec/integration/icon_spec.rb

```diff
@@ -18,6 +18,13 @@
 
           expect(text.font_settings.first[:size]).to eq(60)
         end
+
+        it 'should handle text options (icon height)' do
+          pdf.icon 'far-address-book', size: 60, size_mode: :icon_height
+          text = PDF::Inspector::Text.analyze(pdf.render)
+
+          expect(text.font_settings.first[:size]).to eq(58.25243)
```

Comment:
Nice test! Looks like it's failing on one of the CI matrix executions due to floating point things. Might be able to use the [be_close](https://www.rubydoc.info/gems/rspec-expectations/2.4.0/RSpec%2FMatchers:be_close) matcher to counteract that?

Reply:
Fixed.